src/templates/metatags.jade

```diff
@@ -0,0 +1,19 @@
+<meta property="og:title" content="#{metaTags.title}"/>

```

Comment:
Please change these tags to use jade syntax: `meta(property='og:title' content!=metaTags.title)`